pkg/config/cluster.go

```diff
@@ -39,6 +39,8 @@ const (
 	OpenStackPassword       = "OS_PASSWORD"
 	OpenStackTenantName     = "OS_TENANT_NAME"
 	OpenStackUserName       = "OS_USER_NAME"
+	PacketAPIKey            = "PACKET_API_KEY"
```

Comment:
TODO: make this change for the new API as well. I'll take care of this, but leaving a comment as a reminder.